docs-src/v2/content/audio-input/_index.md

```diff
@@ -166,6 +193,36 @@ print("confidence:", result.confidence)
 // Example coming soon!
 {{< /tab >}}
 
+{{< tab "Swift/iOS" "swift" >}}
+// When there is no more audio data to send
+// (i.e. when the app needs to stop recording audio)
+// call ASRStream's result() method to close the ASR stream
+self.asrStream?.result(completion: { (error) in
+	if let error = error {
+		print(error.localizedDescription)
+	}
+})
+
+// ASR Results come to asrResultHandler block on creating ASRStream
```

Comment:
I'm not sure I understand from the example how the above call to `self.asrStream?.result(...)` relates to this call to create the ASR stream. Swift users must call the result method before creating the stream? Is this a construct that most swift users will understand?

Reply:
No, it's my fault. ASRStream has to be created first. When a developer creates a stream (i.e. when he starts recording, it handles ASR results right in the completion block of this function. 
Then, when recording is stopped, he should call `result` method to close the ASR stream. I will swap these two methods.

Reply:
@mplooster fixed.

Reply:
Thanks!